{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "import csv"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "def split_numbers(numbers):\n",
    "    numbers = numbers.replace(\" \", \"\")\n",
    "    numbers = numbers.split(\"-\")\n",
    "    numbers = [s for s in numbers if s.isdigit()]\n",
    "    return numbers"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['48', '55']"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "split_numbers(\"48-55\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "file = \"legacy_data/main.csv\""
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {
    "collapsed": false,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "with open(file, encoding='utf-8') as csvfile:\n",
    "    data = [x for x in csv.reader(csvfile, delimiter=',') if not x[2].startswith(\"0000\", 0, 4)]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "for x in data[1:]:\n",
    "    temp_ins, _ = Inschrift.objects.get_or_create(legacy_id=x[0])\n",
    "    temp_place, _ = Place.objects.get_or_create(name=x[4].strip())\n",
    "    temp_place.place_type = \"city\"\n",
    "    temp_place.save()\n",
    "    temp_district, _ = Place.objects.get_or_create(name=x[5].strip())\n",
    "    temp_district.place_type = \"city\"\n",
    "    temp_district.part_of = temp_place\n",
    "    temp_district.save()\n",
    "    temp_building, _ = Place.objects.get_or_create(name=x[6].strip())\n",
    "    temp_building.place_type = \"city\"\n",
    "    temp_building.part_of = temp_district\n",
    "    temp_building.save()\n",
    "    temp_building1, _ = Place.objects.get_or_create(name=x[7].strip())\n",
    "    temp_building1.place_type = \"city\"\n",
    "    temp_building1.part_of = temp_building\n",
    "    temp_building1.save()\n",
    "    temp_ins.genauer_standort = temp_building1\n",
    "    temp_ins.alter_standort = x[8].strip()\n",
    "    temp_ins.save()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-08-30 16:18:56) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-08-30 16:19:03) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-08-30 16:19:13) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-08-30 16:50:50) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-08-31 07:47:30) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-08-31 08:48:27) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-09-01 12:35:19) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-09-01 15:34:05) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-09-02 16:26:37) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-09-06 18:36:14) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-10-24 10:02:12) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-11-22 11:07:22) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-11-28 10:47:47) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-06 16:02:03) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-07 15:20:17) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-13 10:28:21) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-13 16:53:46) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-15 09:58:53) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-15 10:51:47) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-15 10:59:04) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-16 11:26:59) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-16 12:18:55) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-16 15:19:09) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-19 11:44:51) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-19 15:12:49) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-20 11:03:25) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-20 11:16:47) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-20 16:24:53) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-21 10:34:02) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-21 11:56:04) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-21 16:58:48) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-22 15:10:07) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-29 12:49:04) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-06 11:00:18) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2011-12-29 14:39:51) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-02 11:12:40) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-02 12:07:16) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-02 15:24:39) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-03 11:12:45) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-03 16:24:06) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-04 16:59:42) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-05 14:22:15) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-05 14:35:12) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-09 16:05:13) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-09 18:22:45) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-10 10:41:19) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-11 11:23:02) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-11 16:18:35) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-12 10:44:28) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-12 14:41:26) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-12 17:13:02) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-13 16:21:42) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-16 18:59:17) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-17 11:56:58) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-17 15:06:29) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-18 15:45:48) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-18 16:16:21) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-18 16:40:53) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-19 16:50:51) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-19 17:02:30) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-24 09:27:12) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-26 08:51:13) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-01-26 15:10:26) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-02-02 09:18:56) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-02-03 09:20:54) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-02-06 15:12:35) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-02-15 17:13:39) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-02-24 14:25:16) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-02-24 15:08:43) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-13 10:06:17) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-19 14:32:28) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-19 14:54:53) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-19 15:15:36) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-19 16:00:58) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-19 16:16:34) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-20 11:56:05) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-21 11:10:41) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-21 13:37:28) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-21 15:33:41) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-21 15:57:55) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-22 08:41:31) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-22 09:11:39) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-22 10:26:01) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-22 14:00:07) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-22 14:33:01) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-03-22 15:08:06) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-08 11:47:37) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-08 12:03:42) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-08 14:54:45) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-08 15:23:11) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-08 16:33:51) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-09 09:51:01) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-09 10:48:39) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-09 14:21:41) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-09 17:00:50) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-10 10:15:22) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-10 10:44:04) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-10 11:00:02) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-10 11:53:39) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-10 13:50:37) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-10 14:15:27) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-10 14:35:35) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-23 10:21:03) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-23 11:05:19) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-25 10:39:38) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-05-25 11:16:47) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-01 11:10:30) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-01 14:44:58) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-06 11:31:55) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-13 14:39:18) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-13 15:16:39) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-14 16:27:32) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-15 10:34:59) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-15 11:20:35) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-23 15:46:24) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-15 12:33:08) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-15 12:51:02) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-19 12:00:08) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-19 14:04:42) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-19 14:32:35) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-19 15:10:31) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-20 13:48:21) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-27 10:15:48) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-06-27 10:57:10) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-20 12:14:40) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-23 16:07:18) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-26 14:42:20) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-26 16:03:07) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-27 16:08:38) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-27 17:39:29) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-31 11:14:56) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-31 11:22:33) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-31 12:59:55) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-31 13:10:28) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-31 13:20:59) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-07-31 16:59:48) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-08-01 11:36:57) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-08-01 16:57:44) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-08-06 12:29:10) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-08-06 13:22:14) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-08-07 10:28:57) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-08-07 11:37:41) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-08-08 15:52:31) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-08-28 18:17:37) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-08-29 13:40:21) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-08-29 14:57:19) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-09-03 13:39:15) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-09-03 14:25:28) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-09-04 10:22:45) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-09-04 12:49:08) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2012-09-04 13:37:46) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-01-31 15:19:32) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-04 12:18:53) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-04 12:19:07) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-05 17:52:13) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-12 17:08:37) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-12 18:33:15) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-13 19:18:19) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-13 19:40:51) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-18 15:36:45) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-18 17:21:56) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-18 18:06:16) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-18 18:43:36) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-19 11:27:53) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-19 14:41:29) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-20 14:29:32) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n",
      "C:\\Users\\pandorfer\\AppData\\Local\\Continuum\\Anaconda3\\envs\\cbab\\lib\\site-packages\\django\\db\\models\\fields\\__init__.py:1453: RuntimeWarning: DateTimeField Inschrift.created received a naive datetime (2013-02-20 16:43:55) while time zone support is active.\n",
      "  RuntimeWarning)\n",
      "\n"
     ]
    }
   ],
   "source": [
    "for x in data[1:]:\n",
    "    temp_ins, _ = Inschrift.objects.get_or_create(legacy_id=x[0])\n",
    "    for y in x[9].split(\";\"):\n",
    "        temp_scheme, _ = SkosConceptScheme.objects.get_or_create(dc_title=\"gattung\")\n",
    "        temp_scheme.save()\n",
    "        temp_concept, _ = SkosConcept.objects.get_or_create(pref_label=y)\n",
    "        temp_concept.save()\n",
    "        temp_concept.scheme.add(temp_scheme)\n",
    "        temp_ins.gattung.add(temp_concept)\n",
    "    for y in x[10].split(\";\"):\n",
    "        temp_scheme, _ = SkosConceptScheme.objects.get_or_create(dc_title=\"traeger\")\n",
    "        temp_scheme.save()\n",
    "        temp_concept, _ = SkosConcept.objects.get_or_create(pref_label=y)\n",
    "        temp_concept.save()\n",
    "        temp_concept.scheme.add(temp_scheme)\n",
    "        temp_ins.traeger.add(temp_concept)\n",
    "    for y in x[38].split(\";\"):\n",
    "        temp_scheme, _ = SkosConceptScheme.objects.get_or_create(dc_title=\"schlagworte\")\n",
    "        temp_scheme.save()\n",
    "        temp_concept, _ = SkosConcept.objects.get_or_create(pref_label=y)\n",
    "        temp_concept.save()\n",
    "        temp_concept.scheme.add(temp_scheme)\n",
    "        temp_ins.schlagworte.add(temp_concept)\n",
    "    for y in x[42].split(\";\"):\n",
    "        temp_scheme, _ = SkosConceptScheme.objects.get_or_create(dc_title=\"schriftklassifikation\")\n",
    "        temp_scheme.save()\n",
    "        temp_concept, _ = SkosConcept.objects.get_or_create(pref_label=y)\n",
    "        temp_concept.save()\n",
    "        temp_concept.scheme.add(temp_scheme)\n",
    "        temp_ins.schlagworte.add(temp_concept)\n",
    "        \n",
    "    temp_ins.farbtraeger = x[17]\n",
    "    temp_ins.farbschrift = x[18]\n",
    "    temp_ins.erhaltungszustand = x[19]\n",
    "    temp_ins.transkription = x[20]\n",
    "    temp_ins.transkription_normalized = x[35]\n",
    "    temp_ins.resch_kopial_signatur = x[24]\n",
    "    temp_ins.resch_kopial_text = x[25]\n",
    "    temp_ins.andere_kopial_signatur = x[26]\n",
    "    temp_ins.anderer_kopial_text = x[27]\n",
    "    temp_ins.notizen = x[28]\n",
    "    temp_ins.bemerkungen = x[29]\n",
    "    temp_ins.quellen_unstruktieriert = x[30]\n",
    "    temp_ins.restaurierungsphasen = x[39]\n",
    "    temp_ins.allgemeine_beschreibung = x[40]\n",
    "    temp_ins.schriftbeschreibung = x[41]\n",
    "    temp_ins.museale_inv_nummer = x[1]\n",
    "    temp_ins.datierung_inschrift_written = x[3]\n",
    "    try:\n",
    "        temp_ins.aufnahmedatum = x[2]\n",
    "    except:\n",
    "        print(x[2])\n",
    "    try:\n",
    "        temp_ins.created = x[31]\n",
    "    except:\n",
    "        print(x[31])\n",
    "    if len(split_numbers(x[11])) == 2:\n",
    "        temp_ins.schrift_hoch_min = split_numbers(x[11])[0]\n",
    "        temp_ins.schrift_hoch_max = split_numbers(x[11])[1]\n",
    "    elif len(split_numbers(x[11])) == 1:\n",
    "        temp_ins.schrift_hoch_min = split_numbers(x[11])[0]\n",
    "        temp_ins.schrift_hoch_max = split_numbers(x[11])[0]\n",
    "    else:\n",
    "        pass\n",
    "    if len(split_numbers(x[12])) == 2:\n",
    "        temp_ins.schrift_breit_min = split_numbers(x[12])[0]\n",
    "        temp_ins.schrift_breit_max = split_numbers(x[12])[1]\n",
    "    elif len(split_numbers(x[12])) == 1:\n",
    "        temp_ins.schrift_breit_min = split_numbers(x[12])[0]\n",
    "        temp_ins.schrift_breit_max = split_numbers(x[12])[0]\n",
    "    else:\n",
    "        pass\n",
    "    if len(split_numbers(x[13])) == 2:\n",
    "        temp_ins.majuskel_min = split_numbers(x[13])[0]\n",
    "        temp_ins.majuskel_max = split_numbers(x[13])[1]\n",
    "    elif len(split_numbers(x[13])) == 1:\n",
    "        temp_ins.majuskel_min = split_numbers(x[13])[0]\n",
    "        temp_ins.majuskel_max = split_numbers(x[13])[0]\n",
    "    else:\n",
    "        pass\n",
    "    if len(split_numbers(x[14])) == 2:\n",
    "        temp_ins.minuskel_min = split_numbers(x[14])[0]\n",
    "        temp_ins.minuskel_max = split_numbers(x[14])[1]\n",
    "    elif len(split_numbers(x[14])) == 1:\n",
    "        temp_ins.minuskel_min = split_numbers(x[14])[0]\n",
    "        temp_ins.minuskel_max = split_numbers(x[14])[0]\n",
    "    else:\n",
    "        pass\n",
    "    if len(split_numbers(x[15])) == 2:\n",
    "        temp_ins.mittellaenge_min = split_numbers(x[15])[0]\n",
    "        temp_ins.mittellaenge_max = split_numbers(x[15])[1]\n",
    "    elif len(split_numbers(x[15])) == 1:\n",
    "        temp_ins.mittellaenge_min = split_numbers(x[15])[0]\n",
    "        temp_ins.mittellaenge_max = split_numbers(x[15])[0]\n",
    "    else:\n",
    "        pass\n",
    "    if len(split_numbers(x[16])) == 2:\n",
    "        temp_ins.oberlaenge_min = split_numbers(x[16])[0]\n",
    "        temp_ins.oberlaenge_max = split_numbers(x[16])[1]\n",
    "    elif len(split_numbers(x[16])) == 1:\n",
    "        temp_ins.oberlaenge_min = split_numbers(x[16])[0]\n",
    "        temp_ins.oberlaenge_max = split_numbers(x[16])[0]\n",
    "    else:\n",
    "        pass\n",
    "    temp_ins.save()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": [
    "#for x in Inschrift.objects.all():\n",
    " #   x.delete()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {
    "collapsed": true,
    "deletable": true,
    "editable": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Django Shell-Plus",
   "language": "python",
   "name": "django_extensions"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.5.2"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
